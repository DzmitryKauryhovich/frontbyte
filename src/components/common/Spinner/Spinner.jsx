import React from 'react'
import { Spin } from 'antd'
import styles from './Spinner.module.scss'

function Spinner() {
  return (
    <div className={styles.root}>
      <Spin className={styles.progress} />
    </div>
  )
}

export default Spinner
