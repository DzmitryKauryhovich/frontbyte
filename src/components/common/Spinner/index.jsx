import React from 'react'
import { Spin } from 'antd'

import styles from './styles.module.scss'

function Spinner() {
  return (
    <div className={styles.root}>
      <Spin className={styles.progress} />
    </div>
  )
}

export default Spinner
