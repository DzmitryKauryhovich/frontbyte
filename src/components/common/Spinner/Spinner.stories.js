import React from 'react'
import Spinner from '.'

export default {
  title: 'Spinner',
  component: Spinner,
}

export const Default = () => (
  <div style={{ width: 300, height: 300, position: 'relative' }}>
    <Spinner />
  </div>
)
