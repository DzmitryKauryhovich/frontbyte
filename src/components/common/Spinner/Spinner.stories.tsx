import React from 'react'
import Spinner from './Spinner'

export default {
  title: 'UI | Spinner',
}

export function story() {
  return <Spinner />
}
