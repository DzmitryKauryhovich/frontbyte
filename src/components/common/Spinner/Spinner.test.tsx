import React from 'react'
import renderer from 'react-test-renderer'
import Spinner from './Spinner'

test('Spinner', () => {
  const component = renderer.create(<Spinner />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
