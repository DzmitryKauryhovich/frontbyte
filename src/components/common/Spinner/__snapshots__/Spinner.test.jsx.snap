// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner 1`] = `
<div
  className="root"
>
  <div
    className="ant-spin ant-spin-spinning progress"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
  </div>
</div>
`;
